### PR TITLE
Enable BDS B2a support for sbp2rinex

### PR DESCRIPTION
**Pull request by dgburr**
Enable B2a for RINEX 3.04 and make this the default output version

### CODE REVIEW

File: src/convrnx.c [hunk 1]
```diff
@@ -101,7 +101,7 @@ static const char vercode[][MAXCODE]={ /* supported obs-type by RINEX version */
     "0........0000..........0000000000...000.............................", /* GAL */
     "2.....22...22..222.....222......2422....................4444........", /* QZS */
     "0......................000..........................................", /* SBS */
-    ".4...4...4.4.....1.......41114..1.....41111............444..44444...", /* BDS */
+    ".4...4...4.4.....1.....4441114..1.....41111............444..44444...", /* BDS */
```
Comment (tao-swift) [inline src/convrnx.c:104]: Do we need to change these two parts? It looks like these two are for I and Q channels, which is not for the BDS L5 B2a signal. The data (D) and pilot (P) channels are given in the second last 444 of this line.

Comment (dgburr) [reply src/convrnx.c:104]: Skylark only sends CODE_BDS3_B5I and the PGM only provides CODE_BDS3_B5Q.

Comment (dgburr) [reply src/convrnx.c:104]: sbp2rinex converts `CODE_BDS3_B5I` to `SYS_CMP`/`CODE_L5I` (24) and `CODE_BDS3_B5Q` to `SYS_CMP`/`CODE_L5Q` (25).

So the lookup table above needs to have valid entries for index 23 and 24 (note that the index is the code number minus 1)

Comment (tao-swift) [reply src/convrnx.c:104]: yeah, that's our naming of calling I and Q channels for the B2a signal, but they are in fact the Data (D) and Pilot (P) channels, respectively, which are specified in index 57 and 58.
Looking into the code, my understand is that RINEX 3.04 doesn't support the use of L5I for BDS B2a signal in Table 9 in http://acc.igs.org/misc/rinex304.pdf.
A better way would be changing these lines https://github.com/swift-nav/RTKLIB/blob/83c8dd2d81a1ba0779ac90ce9648cdc1a2d8d74e/src/rcv/swiftnav.c#L205-L207 to?

```
    [CODE_BDS3_B5I] = {CODE_L5D, SYS_CMP, 2},
    [CODE_BDS3_B5Q] = {CODE_L5P, SYS_CMP, 2},
    [CODE_BDS3_B5X] = {CODE_L5X, SYS_CMP, 2},
```

**Review by tao-swift — APPROVED** [review]

**Pull request merged**